third_party/move/tools/move-mutator/src/report.rs

```diff
@@ -21,19 +21,26 @@ impl Report {
 
     /// Adds a new `MutationReport` to the report.
     pub fn add_entry(&mut self, entry: MutationReport) {
+        trace!("Adding entry to report: {:?}", entry);
         self.mutants.push(entry);
     }
 
     /// Saves the `Report` as a JSON file.
     pub fn save_to_json_file(&self, path: &Path) -> std::io::Result<()> {
         let file = std::fs::File::create(path)?;
+
+        info!("Saving report to {}", path.to_str().unwrap_or(""));
```

Comment:
Maybe:
```suggestion
        info!("Saving report to {}", path.display());
```
Also can be applied below

Reply:
Done.